app/assets/javascripts/darkswarm/services/enterprise_modal.js.coffee

```diff
@@ -2,6 +2,7 @@ angular.module('Darkswarm').factory "EnterpriseModal", ($modal, $rootScope, $htt
   # Build a modal popup for an enterprise.
   new class EnterpriseModal
     open: (enterprise)->
+      return if enterprise.visible == 'hidden'
```

Comment:
I think it's rather unfortunate that this attribute value looks very similar to the css `visibility: hidden`.
It's even more confusing that "Hidden" is not the same as "hidden"!!
https://github.com/openfoodfoundation/openfoodnetwork/blob/7c69cc5aa9abf00f798dbb2809f7308450d37960/config/locales/en.yml#L1002-L1004
(this is not your fault of course!)

I would like to see this part of the system a bit more intuitive. It would be better if the `EnterpriseModel` didn't have to know about the internal values of `Enterprise` and could just call a public method. There's already an `Enterprise#public?` method, so a  new `hide_references?` method would be good.
But I can see this has been waiting three weeks so perhaps we don't need to change this now.

Reply:
Yup why don't we change this later :smile: